resources/charts/stolon/templates/_helpers.tpl

```diff
@@ -0,0 +1,301 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "stolon.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+If release name contains chart name it will be used as a full name.
+*/}}
+{{- define "stolon.fullname" -}}
+{{- if .Values.fullnameOverride -}}
+{{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" -}}
```

Comment:
nit: truncating in the template is at the lowest level - I would assume that that a custom function like `truncName` would be a better choice if the `63` character limitation is removed at some point - or the choice of functions inside a template out of the scope here?

Reply:
it seems using trimSuffix is the de facto standard in Helm charts templates..

eg: https://github.com/helm/charts/blob/master/stable/consul/templates/_helpers.tpl 

or other charts in the stable repo.

Reply:
Even more, `helm create` by default create that helper functions.